package.json

```diff
@@ -64,6 +64,7 @@
     "mathjs": "^5.2.3",
     "prettier": "^1.15.2",
     "rollup": "^0.67.3",
-    "rollup-plugin-babel": "^4.0.3"
+    "rollup-plugin-babel": "^4.0.3",
+    "rollup-plugin-node-resolve": "^3.4.0"
```

Comment:
We can actually remove `rollup-plugin-node-resolve` – it bundles dependencies into the bundle. I think we'd rather just declare our dependencies and dependencies and have npm/yarn handle downloading those. In our case, I guess it's pretty small since we're only doing the lodash snakecase package.

Reply:
@nathunsmitty hmm, I'm pretty sure it just uses the algorithm to resolve and doesn't necessarily include the modules? Just looked over the repo again, but I might be missing something.
https://github.com/rollup/rollup-plugin-node-resolve

Reply:
Turns out we're both right – we need `rollup-plugin-node-resolve` to resolve `index.js` files but it also will try to resolve modules in `node_modules`. To avoid this, we have to use the `externals` option in the rollup config, which we're already doing so I think this is good 👍

Reply:
Hooray! Thanks for digging into this!